upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,19 @@
 == Migration Changes
 
+=== Migrating to 8.0.2
+
+==== More Authentication flows changes
+
+REQUIRED and ALTERNATIVE executions not supported at same flow::
```

Comment:
Maybe `s/REQUIRED and ALTERNATIVE executions not supported at same flow::/REQUIRED and ALTERNATIVE executions not supported at the same level of an authentication flow::/g` ?